### PR TITLE
Added Solution April 1 gfg podt in cpp 

### PR DESCRIPTION
**Pull request by AvinashUmrao**

### CODE REVIEW

**Comment by AvinashUmrao** [comment]
@Ayu-hack sir please , Add hactoberfest label as well

**Comment by Ayu-hack** [comment]
Please share the screenshot and file extension missing.

File: April 2024/Adding solution for Pairs violating the BST property in cpp April 1
Comment (Ayu-hack) [inline April 2024/Adding solution for Pairs violating the BST property in cpp April 1:1]: file extension missing.
what you mean by (in cpp) writing this won't be considered as file extension. learn about file extension and how to use them.

**Comment by Ayu-hack** [comment]
what is gfg (podt)?? 

it should gfg potd !

**Comment by Ayu-hack** [comment]
please check and be careful before requesting a pull

**Comment by Ayu-hack** [comment]
please correct all mistakes and make a new PR

**Pull request closed without merging**